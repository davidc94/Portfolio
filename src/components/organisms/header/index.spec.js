import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('Header component', () => {
  it('render Header as expected', () => {
    const render = renderer.create(
      <Header />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});