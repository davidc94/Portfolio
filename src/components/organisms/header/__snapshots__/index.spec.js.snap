// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render Header as expected 1`] = `
<div
  className="sc-bwzfXH bejfXd"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "flex-start",
    }
  }
  width="full"
>
  <div
    className="sc-bwzfXH iewMrH"
    style={
      Object {
        "flex": 1,
        "paddingLeft": "16px",
      }
    }
  >
    <h1
      className="sc-bdVaJa gSBKap"
      color="#A8DADC"
      size="huge"
    >
      David
    </h1>
    <h1
      className="sc-bdVaJa ksuAVZ"
      color="#A8DADC"
      size="large"
    >
      Chellapah
    </h1>
  </div>
  <div
    className="sc-bwzfXH iewMrH"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <div
      className="sc-bwzfXH iewMrH"
    >
      <h1
        className="sc-bdVaJa jdHjnZ"
        color="#EEE5E9"
        size="base"
      >
        <div
          className=""
        />
      </h1>
    </div>
  </div>
</div>
`;
